### ISSUE TITLE
What does 'deviceType" "scene" mean

### ISSUE THREAD

**Issue by Jhurst4024** [kuestess/homebridge-platform-insteonlocal#232]
Can someone tell me, does 'deviceType" "scene" mean that the property of this "device" is not really an actual device, like a dimmer or switch, but rather is the properties of a scene, such as "On at Sunset" where the properties turn on many lights at sunset or the push of 6/8 button switch?

**Comment by sahafeez** [comment]
@Jhurst4024 It means you are defining a Scene that was configured in the Insteon Hub/PLM. If you created a Scene when you use the application it creates a group with an ID. You can see this in the express server > hub > scene.


For example in my configuration I have this:

```
                {
                    "name": "S.Night",
                    "deviceID": "31.18.75",
                    "dimmable": "yes",
                    "deviceType": "scene",
                    "groupID": "50",
                    "groupMembers": "41.20.3D,41.0C.87"
                },
```

This groupID: reflects the ID group that Insteon picked to put these devices into when you created it in the app, deviceID is the Hub/PLM ID.

**Comment by Jhurst4024** [comment]
Thank you. So all scenes like this in your system would have the "deviceID": "31.18.75" (for your system, my system would have a hub id of ID: 49DD62)? Right?       In your scene, "S.Night", what controls or trigger's it?

**Comment by Jhurst4024** [comment]
Is there a document out there somewhere that has everything you want to know about scenes. Like a Insteon UI "Scenes for Dummies" or "Everything you ever needed to know about Scenes".  Or for that matter something like "Everything you Need to Know about coding for Insteon UI Hub".  LOL

**Comment by sahafeez** [comment]
@Jhurst4024 yes. deviceID is the controller in this case. In my case I am just using Apple Home app. When you create the configure in insteonlocal it just shows up as another device in Home that you can turn on/off. You could just create a "scene" in Home (its scene is not the same as the DB linking that Insteon does between its devices) however @kuestess said the Apple stuff is flakey (which I saw). So I create the scenes in the express server (which says hey, Insteon if you get a msg for group 50 tell its members to take this action) and the use Automation on Apple to schedule them.

If you google Insteon dev guide you will find links. Also you can figure out a ton by first making sure you UNDERSTAND the insteon terms (their docs on linking and unlinking should be read) then just read the wiki here as well as going back and reading closed issues as there is a ton of information in those. Threads like this are helpful.

https://github.com/kuestess/homebridge-platform-insteonlocal/issues/185

**Comment by Jhurst4024** [comment]
Thanks again, very helpful. You are correct, I need to understand the structure of all of the systems to truly get it. I do know some general coding and have done for years, Such as HTML and Javascript. I just don't know the structures of where everything is kept and all of the jargon such as what exactly is Insteonlocal and "express server", etc.

I also want to use Apple Home, but Home is blocking the Hoobs bridge telling me it has already been installed once and I need to Reset it before it will allow me to install it in Home. I have no idea of how to clear it out of Home and let me reinstall the Hoobs Bridge.   If you have any thoughts that would be wonderful.  Thank you again.

**Comment by Jhurst4024** [comment]
The exact message in Apple Home is "Accessory Already Added- To add this accessory to your home, you'll need to reset it first"

**Comment by sahafeez** [comment]
@Jhurst4024 no idea with HOOBS. Are you SURE you removed the bridge from Home? Home>Home Settings>Home Hubs & Bridges? I assume you are talking about the bridge? In HOOBS (do not know this at all) have you deleted the bridge you created? I would nuke the bridge in HOOBS and in Home and see.

**Comment by Jhurst4024** [comment]
What Bridge do you use?  Yes removed from Home. I even nuked Home. It must be in the iCloud somewhere.  I hate to remove from Hoobs because everything is finally set up with about 30 devices. But might have to do.  Here is their website https://hoobs.com

**Comment by sahafeez** [comment]
I just used homebridge directly. 

https://homebridge.io

HOOBS seems to be using part of this and this plugin. That message means the bridge is still in the Home because if you remove the bridge all the devices would go aways. Sorry do not know anymore beyond that. Asking the HOOBS forum might help.

**Comment by kuestess** [comment]
Hi @Jhurst4024 and @sahafeez - converting to a discussion since we now have them :-) as its not specifically an issue.  @Jhurst4024 attached is a nice doc from Insteon that will give a good overview of the tech, groups, links, etc.

[7820933.pdf](https://github.com/kuestess/homebridge-platform-insteonlocal/files/8643228/7820933.pdf)

As @sahafeez said, don't reset your hub, no reason to do so and you risk resetting the password to something other than what is on the bottom of the hub which is bad news.  One thing you may need to do in hoobs is go to the bridges tab, click on the Insteon bridge, and then clear the cache which may reset the pairing and let you add it again.
